modules/heat_transfer/test/tests/heat_conduction/2d_quadrature_gap_heat_transfer/tests

```diff
@@ -76,4 +76,31 @@
     design = "/actions/ThermalContactAction.md"
     issues = "#13043"
   [../]
+
+  [perfect_prereq]
+    type = 'CheckFiles'
+    prereq = perfect
+    input = 'perfect.i'
+    delete_output_before_running = false
+    cli_args = '--split-mesh=2'
+    mesh_mode = 'REPLICATED'
+    check_files = 'perfect.cpa.gz/2/header.gz perfect.cpa.gz/2/split-2-0.gz perfect.cpa.gz/2/split-2-1.gz'
+    recover = false
+    requirement = "The system shall generate a parallel mesh split across 2 processes."
+    design = "/materials/GapConductance.md"
```

Comment:
```suggestion
    design = "Mesh/splitting.md"
```

Reply:
This addressed issues in splitting after all, and this is where the doc was added.

Reply:
yes this is why splitting.md is the better document?

Reply:
Oh yes! But `perfect.i` was already in their test suite and subject to various tests. But sure I can change it .